clientV2/assets/components/_modal.scss

```diff
@@ -0,0 +1,68 @@
+$primaryGray: #777777;
+
+.modal {
+  top: 0;
+  left: 0;
+  height: 100vh;
+  right: 0;
+  background-color: rgba(54, 93, 220, 0.3);
```

Comment:
I think for the sake of uniformity, pls use hex colour code, let reference the color from the theme color that is already created. But if the color we want to use is not there then we can add our hex color to the theme color in assets folder.

Reply:
how can i add opacity while using hex code?

Reply:
This is the hex code for it `#365ddc4d`.
To reveal the hex code, you can click on the rgba text on the color palette when you hover on the `rgba(54, 93, 220, 0.3)` on vscode